backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestOrderRepository.java

```diff
@@ -15,33 +15,38 @@
-				+ "and q.organization.isDeleted = false "
-				+ "and q.patient.isDeleted = false ";
-	public static final String FACILITY_QUERY = BASE_ORG_QUERY + " and q.facility = :facility ";
-	public static final String IS_PENDING = " and q.orderStatus = 'PENDING' "; 
-	public static final String IS_COMPLETED = " and q.orderStatus = 'COMPLETED' ";
+    public static final String BASE_ORG_QUERY = "from #{#entityName} q "
+                + "where q.organization = :org "
+                + "and q.organization.isDeleted = false "
+                + "and q.patient.isDeleted = false ";
+    public static final String FACILITY_QUERY = BASE_ORG_QUERY + " and q.facility = :facility ";
+    public static final String IS_PENDING = " and q.orderStatus = 'PENDING' "; 
+    public static final String IS_COMPLETED = " and q.orderStatus = 'COMPLETED' ";
+    public static final String HAS_EVENT = " and q.testEvent IS NOT NULL ";
     public static final String ORDER_CREATION_ORDER = " order by q.createdAt ";
     public static final String RESULT_RECENT_ORDER = " order by q.testEvent.createdAt desc ";
 
     @Query(FACILITY_QUERY + IS_PENDING + ORDER_CREATION_ORDER)
-	@EntityGraph(attributePaths = "patient")
-	public List<TestOrder> fetchQueue(Organization org, Facility facility);
+    @EntityGraph(attributePaths = "patient")
+    public List<TestOrder> fetchQueue(Organization org, Facility facility);
 
-	@Query(BASE_ORG_QUERY + IS_PENDING + " and q.patient = :patient")
-	@EntityGraph(attributePaths = "patient")
-	public Optional<TestOrder> fetchQueueItem(Organization org, Person patient);
+    @Query(BASE_ORG_QUERY + IS_PENDING + " and q.patient = :patient")
+    @EntityGraph(attributePaths = "patient")
+    public Optional<TestOrder> fetchQueueItem(Organization org, Person patient);
 
-	@Query(BASE_ORG_QUERY + IS_PENDING + " and q.id = :id")
-	public Optional<TestOrder> fetchQueueItemById(Organization org, UUID id);
+    @Query(BASE_ORG_QUERY + IS_PENDING + " and q.id = :id")
+    public Optional<TestOrder> fetchQueueItemById(Organization org, UUID id);
 
     @Query(FACILITY_QUERY + IS_COMPLETED + RESULT_RECENT_ORDER)
-	@EntityGraph(attributePaths = "patient")
-	public List<TestOrder> fetchPastResults(Organization org, Facility facility);
+    @EntityGraph(attributePaths = "patient")
+    public List<TestOrder> fetchPastResults(Organization org, Facility facility);
 
-	@Query("update #{#entityName} q set q.orderStatus = 'CANCELED' "
-			+ "where q.organization = :org and q.orderStatus = 'PENDING'")
-	@Modifying
-	public int cancelAllPendingOrders(Organization org);
+    @Query("update #{#entityName} q set q.orderStatus = 'CANCELED' "
+            + "where q.organization = :org and q.orderStatus = 'PENDING'")
+    @Modifying
+    public int cancelAllPendingOrders(Organization org);
+
+    @Query(FACILITY_QUERY + IS_COMPLETED + HAS_EVENT)
+    @EntityGraph(attributePaths = "testEvent")
+    public List<TestOrder> getTestResults(Organization org, Facility facility);
```

Comment:
Why not just add the EntityGraph annotation to the method in TestEventRepository?